doc/src/manual/strings.md

```diff
@@ -348,6 +348,35 @@ x
 y
 ```
 
+Strings in Julia can contain invalid UTF-8 code unit sequences. This convention allows to
+treat any byte sequence as a `String`. In such situations a rule is that characters are formed
+by longest possibly valid sequences of code points. This rule is best explained by an example:
```

Comment:
"possibly valid" isn't very explicit.

Reply:
I have tried to improve it (but it is hard).

Reply:
Yeah. The new version much more precise, but is "the longest sequence of code units that could be a start of some valid code point" really correct? e.g. an overlong encoding isn't a start of a valid character. Sorry, I don't know what the best description could be, but Stefan can probably help.

Reply:
I have made another shot (more verbose) and with one additional example.